tests/pytest/run_files/job_jenkins_sarus

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -l
```

Comment:
Putting this script here gives us a certain amount of technical dept, since we can no longer centrally make changes to the SLURM scripts we use on Piz Daint (e.g. changing `--account`) since these scripts are scattered all over the place. Not sure what the right solution is, but we could `curl https://raw.githubusercontent.com/VulcanClimateModeling/buildenv/master/submit.daint.slurm` instead to get this file.

Reply:
There is a trade-off between DRY (do not repeat yourself) and being explicit. I think it's important in these tests to have the full description of what the test is doing included in the repo. Otherwise, re-running an earlier test can produce a different result if e.g. `buildenv/master/submit.daint.slurm` changes. It isn't ideal, but it is very possible to edit a file in 5 repositories, so the downside in that case should be weighed against the downside of having non-reproducible tests.

For a detail we expect to regularly change like `--account`, is there a way we can parametrize it using an environment variable or user setting in Jenkins?

Reply:
Issue for a future PR, I believe

Reply:
Ok.